### PR TITLE
Update documentation

### LINKED ISSUE #163: Add tool description to README
There are short descriptions of demos and projects. We should also have short descriptions of tools.

### PR DESCRIPTION
**Pull request by mickvangelderen**
Fixes #163 and #171

### CODE REVIEW

File: projects/flimnap/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Runs static simulator code interfacing with Unity environment
+Bikesim(5ade352 or newer). Cannot log data via SDIO.
```
Comment (oliverlee) [inline projects/flimnap/README.md]: This should point to phobos-visualizer.

**Comment by oliverlee** [comment]
I'm not sure this is the best approach. Before we had a quick summary of the binaries which you could look at the root level README. Now you need to descend down multiple levels to get a brief description.

**Comment by mickvangelderen** [comment]
> I'm not sure this is the best approach. Before we had a quick summary of the binaries which you could look at the root level README. Now you need to descend down multiple levels to get a brief description.

There is a balance I guess. I like localized readmes right now, it encourages exploration of the repository. It also makes it slightly easier to keep documentation up to date because it is closer to the subject. 

The single line descriptions of some of the projects do not do them justice. 

I don't mind taking either approach or a combination.

**Comment by oliverlee** [comment]
I like the idea of having single line descriptions at the root level and
more detailed descriptions in the localized readmes.

On Thu, May 11, 2017 at 9:32 AM, Mick van Gelderen <notifications@github.com

**Comment by oliverlee** [comment]
Opened new issue for improving localized READMEs: https://github.com/oliverlee/phobos/issues/182

**Pull request merged**